src/Monitor.cs

```diff
@@ -181,6 +181,12 @@ public async Task StartCaptureAsync(string networkInterface, CancellationToken c
                 {
                     _device = devices.Where(x => 
                         x.Name.Equals(networkInterface, StringComparison.InvariantCultureIgnoreCase)).FirstOrDefault();
+
+                    if (_device == null) // still unresolved, search by friendly name (ie. ipconfig /all)
+                    {
+                        _device = devices.Where(x => x.Interface.FriendlyName != null &&
+                            x.Interface.FriendlyName.Equals(networkInterface, StringComparison.InvariantCultureIgnoreCase)).FirstOrDefault();
+                    }
```

Comment:
In the spirit of consistency, can you refactor the 2 LINQ .Where() clauses here to simply use .FirstOrDefault(x => ...). Having Where(x => ...).FirstOrDefault() is somewhat redundant. I realize that I introduced that inconsistency in line 182, but this seems like a good opportunity to address that.